apps/snitch_api/lib/snitch_api/products_context/products_context.ex

```diff
@@ -7,19 +7,22 @@ defmodule SnitchApi.ProductsContext do
 
   import Ecto.Query, only: [from: 2, order_by: 2]
 
+  @filter_allowables ~w(taxon_id brand_id is_active)a
+  @partial_search_allowables ~w(name slug)a
```

Comment:
Remove `is_active` and `slug` filters.
We only want following filters `name`, `taxon_id`, `brand_id`